pkg/controller/clusterautoscaler/validator.go

```diff
@@ -136,6 +137,13 @@ func (v *Validator) validateScaleDownConfig(sd *autoscalingv1.ScaleDownConfig) u
 			errs = append(errs, fmt.Errorf("ScaleDown.%s: %v", name, err))
 		}
 	}
+	utilizationThreshold, err := strconv.ParseFloat(*sd.UtilizationThreshold, 64)
+	if err != nil {
+		errs = append(errs, errors.New("ScaleDown.UtilizationThreshold must be a string representing float value."))
+	}
+	if utilizationThreshold <= float64(0) || utilizationThreshold >= float64(1) {
+		errs = append(errs, errors.New("ScaleDown.UtilizationThreshold must be a value between 0 and 1."))
+	}
```

Comment:
i think we should add a unit test to exercise this validation

Reply:
@SamuelStuchly would you mind adding a unit test for this?

Reply:
Will do.